ros2pkg/ros2pkg/build_type/__init__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Wayne Parrott
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.entry_points import load_entry_points
+from ros2cli.plugin_system import instantiate_extensions
+from ros2cli.plugin_system import PLUGIN_SYSTEM_VERSION
+from ros2cli.plugin_system import satisfies_version
+
+
+class BuildTypeExtension:
+    """
+    The interface for verb extensions.
+
+    The following properties must be defined:
+    * `NAME` (will be set to the entry point name)
+
+    The following methods must be defined:
+    * `main`
+    """
+
+    NAME = None
+    EXTENSION_POINT_VERSION = '0.1'
```

Comment:
`EXTENSION_POINT_VERSION` doesn't seem to be used anywhere. Do we need it?

Reply:
I followed the pattern of all the rospkg extension points which include EXTENSION_POINT_VERSION. I scanned the codebase and did not find functional references to it. My assumption is the original developer had planned to make use of it at future date. I've elected to retain it in the build-type extension point for consistency with other extension points.